lbz/exceptions.py

```diff
@@ -1,7 +1,5 @@
 # coding=utf-8
-"""
-Set of HTTP Exceptions that are json compatabile.
```

Comment:
_FYI:_ I have removed this docstring as this does not conform to all exceptions being declared here.